backend/elasticsearch/src/main/java/org/hibernate/search/backend/elasticsearch/search/query/impl/ElasticsearchSearchQueryImpl.java

```diff
@@ -149,7 +146,21 @@ public long fetchTotalHitCount() {
 
 	@Override
 	public SearchScroll<H> scroll(Integer pageSize) {
-		throw new UnsupportedOperationException( "Not yet implemented" );
+		NonBulkableWork<ElasticsearchLoadableSearchResult<H>> work = getSearchWorkBuilder()
+				.scrolling( pageSize, "60s" )
```

Comment:
This hardcoded value here needs a TODO :)

Reply:
right! I didn't put the TODO, anyway the row is overwritten by a further commit.